<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>294</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>425</width>
    <height>294</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>425</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>批量栅格数据转换器</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>制作者：陨星落云   联系邮箱：672319707@qq.com</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>176</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>输出数据格式：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="dataFormat_comboBox">
      <item>
       <property name="text">
        <string>.tif</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>.bil</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>.img</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>.png</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>.ntf</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>.pix</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>输出数据类型：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="dataType_comboBox">
      <item>
       <property name="text">
        <string>UInt16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Byte</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Int16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>UInt32</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Int32</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Float32</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FLoat64</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>370</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>输出文件夹：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item>
     <widget class="QPushButton" name="output_pushButton">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>370</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>输入文件：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="input_pushButton">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>欢迎关注公众号：测绘数据分析</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>QT_CMake/out/build/x64-Debug/images/wechat.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
